tests/testthat/test_pipelines.R

```diff
@@ -151,3 +151,40 @@ test_that("survtoclassif_disctime", {
   # model with more covariates should have better C-index
   expect_gt(pred2$score(), pred$score())
 })
+
+skip_if_not_installed("mlr3extralearners")
+
+test_that("survtoclassif_IPCW", {
+  requireNamespace("mlr3extralearners")
```

Comment:
better not use `mlr3extralearners` as it is not on CRAN and in the future I might change it back to use an `mlr3learner`